packages/core/src/transform/template/template-to-typescript.ts

```diff
@@ -42,7 +42,7 @@ export function templateToTypescript(
   return mapTemplateContents(originalTemplate, { embeddingSyntax }, (ast, mapper) => {
     let { emit, record, rangeForLine, rangeForNode } = mapper;
     let scope = new ScopeStack([]);
-
+    let inSVG = false;
```

Comment:
it could be possible to have components inside svg, with this it would be possible to a glint-directive `@glint-in-svg` a the top of the template to specify that we are inside svg.